docker/Dockerfile

```diff
@@ -1,5 +1,8 @@
 # SPDX-License-Identifier: GPL-3.0+
-FROM alpine:edge as builder
+# author  : paulfantom
+
+ARG ARCH=amd64-v3.8
```

Comment:
Probably worth adding a comment here that this is overridden in the default build job by `build.sh`. I can see someone in the future bumping this version and no knowing that is is never used in the normal process of publishing the images

Reply:
True, I'll add that.